src/GraphQL/Execution/ExecutionOptions.cs

```diff
@@ -12,22 +12,42 @@
 
 namespace GraphQL
 {
+    /// <summary>Configuration options to be passed to <see cref="IDocumentExecuter"/> to execute a query</summary>
     public class ExecutionOptions : IProvideUserContext
     {
+        /// <summary>Schema of graph to use; required<br/><br/>Schema will be initialized if has not yet been initialized.</summary>
         public ISchema Schema { get; set; }
+
+        /// <summary>Object to pass to the <see cref="IResolveFieldContext.Source"/> property of first-level resolvers</summary>
         public object Root { get; set; }
+
+        /// <summary>GraphQL query to parse and execute; required</summary>
         public string Query { get; set; }
+
+        /// <summary>GraphQL query operation name; optional, defaults to first (if any) operation defined in query</summary>
         public string OperationName { get; set; }
+
+        /// <summary>
+        /// Parsed GraphQL request; will parse <see cref="Query"/> if not set.<br/>
+        /// <br/>
+        /// Can be used when implementing a cache of parsed GraphQL requests (a <see cref="Language.AST.Document"/>)</summary>
```

Comment:
```suggestion
        /// Can be used to increase performance when implementing a cache of parsed GraphQL requests (a <see cref="Language.AST.Document"/>). If not set, it will be parsed from <see cref="Query"/></summary>
```

Reply:
Just to clarify, are you suggesting I remove the first line also?

So:
```
/// <summary>Can be used to increase performance when implementing a cache of parsed GraphQL requests (a <see cref="Language.AST.Document"/>). If not set, it will be parsed from <see cref="Query"/></summary>
```

Reply:
```
/// <summary>Parsed GraphQL request; can be used to increase performance when implementing a cache of parsed GraphQL requests (a <see cref="Language.AST.Document"/>). If not set, it will be parsed from <see cref="Query"/></summary>
```

Reply:
Looks good. Updated as suggested.